chrome/browser/android/adblock/regex_resource_type_detector.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * This file is part of Adblock Plus <https://adblockplus.org/>,
+ * Copyright (C) 2006-present eyeo GmbH
+ *
+ * Adblock Plus is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * Adblock Plus is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Adblock Plus.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef CHROME_BROWSER_ANDROID_REGEX_RESOURCE_TYPE_DETECTOR_H_
+#define CHROME_BROWSER_ANDROID_REGEX_RESOURCE_TYPE_DETECTOR_H_
```

Comment:
BTW, perhaps it's for consitency, but IMO it would be much better to use `#pragma once`.

Reply:
https://chromium.googlesource.com/chromium/src/+/master/styleguide/c++/c++.md

> Do not use #pragma once; historically it was not supported on all platforms, and it does not seem to outperform #include guards even on platforms which do support it.

Reply:
OK, I still think `#pragma once` is better, though, but let's not use it here.

Reply:
I'm split between staying consistent with Chrome style guide, and using proper modern C++. 

I believe that none of the compiler we use for Android is bad enough to not support `#pragma once`, which is less error prone.